processor/groupbytraceprocessor/storage_memory.go

```diff
@@ -25,7 +25,7 @@ import (
 
 type memoryStorage struct {
 	sync.RWMutex
-	content                   map[string][]pdata.ResourceSpans
+	content                   map[[16]byte][]pdata.ResourceSpans
```

Comment:
You're right, but I don't like this idea, because in case of changes in the model it will implicitly affect this processor. Current approach guarantees correctness

Reply:
Pretty much every processor and exporter that needs to deal with TraceID uses `pdata.TraceId`, @bogdandrutu usually writes scripts that updates those places en masse.